charts/immich/values.yaml

```diff
@@ -8,207 +8,119 @@
-            enabled: true
-            custom: true
-            spec:
-              httpGet:
-                path: /ping
-                port: http
-              initialDelaySeconds: 0
-              periodSeconds: 10
-              timeoutSeconds: 1
-              failureThreshold: 3
-          startup:
-            enabled: true
-            custom: true
-            spec:
-              httpGet:
-                path: /ping
-                port: http
-              initialDelaySeconds: 0
-              periodSeconds: 10
-              timeoutSeconds: 1
-              failureThreshold: 60
+      main: {}
+        # resources:
+        #   limits:
+        #     nvidia.com/gpu.shared: '1'
 
-podAnnotations:
-  checksum/config: "{{ .Values.immich.configuration | toYaml | sha256sum }}"
+# see https://bjw-s-labs.github.io/helm-charts/docs/common-library/storage/types/persistentVolumeClaim/#existing-claim
+persistence:
+   library:
+     accessMode: ReadWriteOnce
+     size: 100Gi
+     nameOverride: ''
+     storageClass: ''
+   # Optional: Set this to pvc to avoid downloading the ML models every start.
+   # machine-learning-cache:
+   #   enabled: true
+   #   accessMode: ReadWriteOnce
+   #   size: 10Gi
+   # external-1:
+   #   type: persistentVolumeClaim
+   #   existingClaim: ''
+   #   advancedMounts:
+   #     server:
+   #       main:
+   #         - path: /usr/src/app/external-1
 
-serviceMonitor:
-  server:
-    serviceName: server
-    endpoints:
-      - port: metrics-api
-        scheme: http
-      - port: metrics-ms
-        scheme: http
+# ingress:
+#   main:
+#     annotations: {}
+#       # proxy-body-size is set to 0 to remove the body limit on file uploads
+#       # nginx.ingress.kubernetes.io/proxy-body-size: "0"
+#     hosts:
+#       - host: immich.local
+#         paths:
+#           - path: "/"
+#             service:
+#               identifier: server
+#     tls: []
 
-# persistence:
-#   server:
-#     identifier: server
-#     advancedMounts:
-#       - type: configMap
-#         name: "{{ .Release.Name }}-immich-config"
-#       - name: immich-library
-#         path: /usr/src/app/upload
-#         accessMode: ReadWriteMany
-#         size: 1Ti
-#         storageClass: # provide your storage class
-#         # existingClaim: immich-library Provide your existing PVC
-#       - path: /usr/src/app/external
-#         name: immich-external
-#         accessMode: ReadWriteMany
-#         size: 1Ti
-#         storageClass: # provide your storage class
-#         # existingClaim: immich-external Provide your existing PVC
-#   machine-learning:
-#     identifier: machine-learning
-#     advancedMounts:
-#       - name: cache
-#         mountPath: /cache
-#         size: 10Gi
-#         # Optional: Set this to pvc to avoid downloading the ML models every start.
-#         type: emptyDir
-#         accessMode: ReadWriteMany
-#         storageClass: # provide your storage class
+rawResources:
+  psql:
+    enabled: false
```

Comment:
maybe we could share credentials between both postgres definitions (use the same secret, or value)

In any case, we should document the options one has, and ensure that not both are activated at the same time?

Reply:
it is shared. the user would set the `DB_PASSWORD` to reference the same secret in values.
``` yaml
immich:
  env:
    DB_PASSWORD:
      valueFrom:
        secretKeyRef:
          name: immich-psql
          key: password

```